pkg/kudoctl/cmd/init_integration_test.go

```diff
@@ -145,7 +145,7 @@ func TestIntegInitWithNameSpace(t *testing.T) {
 	// On first attempt, the namespace does not exist, so the error is expected.
 	err = cmd.run()
 	require.Error(t, err)
-	assert.Equal(t, "error installing: Namespace integration-test does not exist - KUDO expects that any namespace except the default kudo-system is created beforehand\n", err.Error())
+	assert.Equal(t, "error installing Namespace integration-test does not exist - KUDO expects that any namespace except the default kudo-system is created beforehand\n", err.Error())
```

Comment:
These sentences do not make sense now. E.g. removing the colon here makes it seem like there was "an error while installing a namespace" but then "does not exist" appears which makes the logic shatter.

Reply:
You're right. Hm, I don't see what `error installing` prefix adds to the error. I'll remove it altogether.